.testing/Makefile

```diff
@@ -88,6 +88,13 @@ SOURCE = $(wildcard $(BASE)/src/*/*.F90 $(BASE)/src/*/*/*.F90 $(BASE)/config_src
 all: $(foreach b,$(BUILDS),$(BUILD)/$(b)/MOM6)
```

Comment:
Might as well edit this to use the rules defined below.

Reply:
I wasn't meaning to change the live rules, just add some shortcuts/